superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -1509,6 +1511,7 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
       show={show}
       title={<h4>{t('Connect a database')}</h4>}
       footer={renderModalFooter()}
```

Comment:
@AAfghahi Do you want to be using the modalFooter variable here?

Reply:
This is referencing line 1511? If so, no I don't think so becuase it comes after a ternary that checks for isEditMode